test/generator/test_generator_reach.py

```diff
@@ -100,6 +100,9 @@ def _compare_actions(first_reach: GeneratorReach,
     # These 3 indices are in Olympus and are unreachable given the default preset
     RandovaniaGame.METROID_PRIME_CORRUPTION: {0, 1, 2},
 
+    # These items require shinesparking, which due to varying difficulty has been made into a trick
```

Comment:
I'd say it'd be nice to run the check with a few tricks then, maybe? Hm

Reply:
Would be nice, yeah.
Not quite sure how to implement though. I wanted to avoid modifying the starter preset, as there are quite a few who don't have that level of shinesparking and I didn't want to exclude them so I wanted to opt for a "include tricks when checking for all pickups" approach. However, I'm a bit lost on what actually excludes the tricks in the first place, as far as I can see they're already included [here](https://github.com/randovania/randovania/blob/main/test/generator/test_generator_reach.py#L129). Probably something in reach_lib?

Reply:
You can change the preset dynamically during the preset. This example for a Dread test:
```python
    gd = default_database.game_description_for(preset.game)
    suitless = gd.resource_database.get_by_type_and_index(ResourceType.TRICK, "Suitless")

    configuration = dataclasses.replace(
        configuration,
        trick_level=configuration.trick_level.set_level_for_trick(suitless, LayoutTrickLevel.HYPERMODE),
    )
```

Could be a new set of overrides at the top or something.